script/aws_sts.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Requesting a token code from the user
+read -p "Enter token code: " token_code
+
+# Command execution aws sts assume-role с подстановкой значения кода токена
+output_aws=$(aws sts assume-role --role-arn arn:aws:iam::<AccountID>:role/Admins --role-session-name tf --serial-number <your_mfa_id> --token-code "$token_code" --profile <your_profile>)
```

Comment:
hmm  i think we need to be able to pass `profile` name to script  and get `--serial-number --role-arn` from profile information. Something like 
`aws configure  --profile dos14 get role_arn`

Reply:
By default, we cannot get values ​​from `aws configre get` until we write them there via `aws configure set`.  I can make it so that when the script is executed, a separate profile is created in which these values ​​​​are written and any prompts is no longer needed.